pkg/dependencies/factory.go

```diff
@@ -1317,6 +1317,29 @@ func (f *Factory) WithPackageInstaller(spec *cluster.Spec, packagesLocation, kub
 	return f
 }
 
+// WithPackageInstallerWithoutWait builds a package installer that doesn't wait for active bundles.
+func (f *Factory) WithPackageInstallerWithoutWait(spec *cluster.Spec, packagesLocation, kubeConfig string) *Factory {
+	f.WithKubectl().WithPackageControllerClientWithoutWait(spec, kubeConfig).WithPackageClient()
+	f.buildSteps = append(f.buildSteps, func(_ context.Context) error {
```

Comment:
Can we avoid having this repeated code? can we just make use of the original builder method from here? like defining it as a dependency